dcos/package.py

```diff
@@ -943,6 +944,25 @@ def copy_to_cache(self, target_dir):
                 'Unable to fetch packages from [{}]'.format(self.url))
 
 
+def onerror(func, path, exc_info):
```

Comment:
please follow the pydoc documentation format we use for all other functions.  Also, if this function is private to this module, prefix it with a `_`